src/containers/ClassifierContainer.jsx

```diff
@@ -98,6 +120,62 @@ class ClassifierContainer extends React.Component {
   }
 
   render() {
+    //JULY UPDATE:
+    //Allow users to select their workflow
+    //----------------------------------------------------------------
+    const startWorkflow = (workflow_id, variant_type) => {
+      this.props.dispatch(fetchWorkflow(workflow_id)).then(() => {
+        this.props.dispatch(fetchSubject());
+        this.props.dispatch(setVariant(variant_type));
+      });
+    }
+    
+    if (this.props.workflowStatus === WORKFLOW_STATUS.IDLE) {
+      return (
+        <main className="app-content flex-column flex-center">
+          <p>CHOOSE YOUR WORKFLOW</p>
+          <div>
+            <button onClick={() => { startWorkflow(config.zooniverseLinks.workflowId, VARIANT_TYPES.INDIVIDUAL) }}>
+              Solo
+            </button>
+            <button onClick={() => { startWorkflow(config.zooniverseLinks.collabWorkflowId, VARIANT_TYPES.COLLABORATIVE) }}>
+              Collaborative
+            </button>
+          </div>
+          
+          {(this.state.popup === null) ? null :
+            <Popup onClose={this.closePopup.bind(this)}>
+              {this.state.popup}
+            </Popup>
+          }
+          
+          {this.renderSignInReminder()}
+        </main>
+      );
+    }
+    //----------------------------------------------------------------
+    
+    //Status Checks
+    //----------------------------------------------------------------
+    if (this.props.workflowStatus === WORKFLOW_STATUS.FETCHING) {
+      return (
+        <main className="app-content flex-column flex-center">
+          LOADING...
```

Comment:
I see the `LoadingSpinner` was added to this component. Was that to be added here?